internal/provider/resource_aws_network_peering_test.go

```diff
@@ -14,7 +13,6 @@ import (
 
 var (
 	// using unique names for AWS resource to make debugging easier
-	hvnPeeringUniqueAWSName = fmt.Sprintf("hcp-tf-provider-test-%d", rand.Intn(99999))
```

Comment:
Might need to rebase this on my [variable change PR](https://github.com/hashicorp/terraform-provider-hcp/pull/428)